services/azfile/internal/cmd/meta.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	def "go.beyondstorage.io/v5/definitions"
+	"go.beyondstorage.io/v5/types"
+)
+
+var Metadata = def.Metadata{
+	Name: "azfile",
+	Infos: []def.Info{
+		infoObjectMetaServerEncrypted,
+	},
+	Pairs: []def.Pair{},
+	Factory: []def.Pair{
+		def.PairCredential,
+		def.PairEndpoint,
+		def.PairName,
+		def.PairWorkDir,
+	},
+	Service: def.Service{
```

Comment:
If we don't support the service, please don't assign this service.

Reply:
https://github.com/Joey2132/go-storage/blob/master/definitions/metadata.go#L38
If we don't assign it, the make build will fail with `panic: runtime error: invalid memory address or nil pointer dereference `.

Reply:
Please report a bug to go-storage before workaround it.

Reply:
Got it.